include/deal.II/lac/affine_constraints.h

```diff
@@ -2756,7 +2756,7 @@ AffineConstraints<number>::copy_from(
   lines.clear();
   lines.reserve(other.lines.size());
 
-  for (const auto l : other.lines)
+  for (const auto &l : other.lines)
```

Comment:
This is unrelated, or does it have to do with the other PR? (I don't think we need to be overly formal and open another PR, I'll approve it with this).